doc/ceph-executive-council-responsibilities.rst

```diff
@@ -0,0 +1,120 @@
+.. _ceph-executive-council-responsibilities:
+
+Ceph Executive Council Responsibilities
+=======================================
+
+.. warning:: It is forbidden to change the text of this document without the
+   approval of the Ceph Steering Committee.
+
+
+Old state
+
+- Sage: https://pad.ceph.com/p/sage-responsibilities
```

Comment:
I dislike the durability of etherpad. I wonder if there is a way we can archive this sensibly.